content/articles/generic-programming-in-c#/index.md

```diff
@@ -0,0 +1,182 @@
+
+### Introduction
+
+In any modern system, data handling is one of the major and vital goals. The way data is handled is what makes the total difference. Some systems go for generic one which includes lists, dictionaries, and hash.
```

Comment:
> Some systems go for generic one which includes lists, dictionaries, and hash.

Change this to:

> Some systems go for a generic approach which includes the use of lists, dictionaries, and hash.

To be more grammatically correct and specific. Also, I was a little confused about the meaning of *hash* in this context because that word can have [many meanings](https://cs.stackexchange.com/a/55472). So you may need to clarify or reword it a bit here.

Reply:
> Some systems go for generic one which includes lists, dictionaries, and hash.
> 
> Others prefer non-generic which are arrays, array lists, and bit arrays.

These two sentences are closely related to each other and should be in the same paragraph.